Packages/com.unity.inputsystem/InputSystem/Controls/InputControlExtensions.cs

```diff
@@ -1083,9 +1083,9 @@ public static bool HasButtonPress(this InputEventPtr eventPtr, float magnitude =
         /// <summary>
         /// Get the first pressed button from the given event or null if the event doesn't contain a new button press.
         /// </summary>
-        /// <param name="eventPtr"></param>
-        /// <param name="magnitude"></param>
-        /// <param name="buttonControlsOnly"></param>
+        /// <param name="eventPtr">An event. Must be a <see cref="StateEvent"/> or <see cref="DeltaStateEvent"/>.</param>
```

Comment:
Seeing now that there is actually a exception doc for this. Let's remove the `ArgumentException` doc RE `StateEvent`/`DeltaStateEvent` and add something here that states that if the event is not of this type, the method will always return `null`.